### PR TITLE
adding new pairs and delete last tr

### PR DESCRIPTION
**Pull request by nemanja777**
We have implemented adding a new pair(now it works without refresh), but the last pair cannot be deleted without refresh. The task is in the TODO comment.

### CODE REVIEW

File: public/js/santaAdmin.js [hunk 1]
```diff
@@ -81,14 +82,15 @@ $(async function() {
...
     });
   });
-
   $('#forbiddenPairsForm').on('submit', () => {
     const pair = {
       forbiddenUser1Id: $('#forbiddenUser1').val(),
       forbiddenUser2Id: $('#forbiddenUser2').val()
     };
     $.post('api/forbidden', pair, result => {
-      showAlert(result);
```
Comment (Zaista) [inline public/js/santaAdmin.js:91]: Why did you remove this?

Comment (nemanja777) [reply public/js/santaAdmin.js:91]: Probably trying to get the code right, I don't have a specific reason. This is a mistake.
